.github/workflows/ca-basic-test.yml

```diff
@@ -52,6 +52,7 @@ jobs:
               -f /usr/share/pki/server/examples/installation/ca.cfg \
               -s CA \
               -D pki_ds_url=ldap://ds.example.com:3389 \
+              -D pki_audit_signing_nickname= \
```

Comment:
This is the default value, is there a reason to add it for the installation?

Reply:
Yes, that's because we define the `pki_audit_signing_nickname` in the `ca.cfg` example that we ship so it needs to be overridden with a blank value:
https://github.com/dogtagpki/pki/blob/master/base/server/examples/installation/ca.cfg

We're currently in the middle of updating the docs which might include these config files as well so I'm trying to avoid changing these files right now. Later once the doc update is complete we can clean up the tests and the examples again.